app/utils/callback/lava.py

```diff
@@ -510,7 +510,10 @@ def _add_rootfs_info(group, base_path, file_name="build_info.json"):
         return
 
     try:
-        if rootfs_url.startswith(DEFAULT_STORAGE_URL):
+        # compare to default URL without the scheme
+        _default_url = urllib2.urlparse.urlparse(DEFAULT_STORAGE_URL).netloc
```

Comment:
This could have been set as a fixed module-wide string to avoid parsing it every time, but it's OK for now.